app/assets/javascripts/channels/comment.coffee

```diff
@@ -0,0 +1,53 @@
+App.comment = App.cable.subscriptions.create "CommentChannel",
+  connected: ->
+    # Called when the subscription is ready for use on the server
+
+  disconnected: ->
+    # Called when the subscription has been terminated by the server
+
+
+  # Called when there's incoming data on the websocket for this channel
+  received: (data) ->
+    event = new CommentEvent(data)
+    event.process() if event.isRelevant()
+
+
+class CommentEvent
+  constructor: (data) ->
+    @$feed     = $("[data-commentable='#{data.comment_feed_id}']")
+    @action    = data.action
+    @commentId = data.comment_id
+    @html      = data.html
+
+  # We need to make sure that the comment we've been notified about belongs
+  # on the currently-viewed page
+  isRelevant: ->
+    @$feed.length != 0
+
+  toggleActions: ->
```

Comment:
we should find a way to skip this. If the user disables js or changes its html via browser console, he may see the forbidden links?

Reply:
That's the point - I know this is a crappy solution, but it's the best I could come up given my limited understanding of ActionCable.

Worth noting that even if the user can *see* links he shouldn't, they won't work because the user will be validated on the server-side (or at least my understanding is that this is being handled by `cancancan`.) That's probably worth writing a request spec for in any case.